website/docs/r/alert_channel_gcp_pub_sub.html.markdown

```diff
@@ -16,9 +16,10 @@ To find more information see the [Lacework support documentation](https://suppor
 
 ```hcl
 resource "lacework_alert_channel_gcp_pub_sub" "example" {
-  name       = "gcp-pub_sub"
-  project_id = "lacework-191923"
-  topic_id   = "lacework-alerts"
+  name            = "gcp-pub_sub"
+  project_id      = "lacework-191923"
+  topic_id        = "lacework-alerts"
+  issue_grouping  = "Events"
```

Comment:
I wonder if we need to add this to the example... 🤔 Since it is optional, maybe not adding it would be better